setup.py

```diff
@@ -30,8 +32,83 @@
 from distutils.command.build_scripts import build_scripts as BuildScripts
 from distutils.command.sdist import sdist as SDist
 
-sys.path.insert(0, os.path.abspath('lib'))
-from ansible.release import __version__, __author__
+
+def find_package_info(*file_paths):
+    # Open in Latin-1 so that we avoid encoding errors.
+    # Use codecs.open for Python 2 compatibility
+    try:
+        with codecs.open(os.path.join(*file_paths), 'r', 'latin1') as f:
+            info_file = f.read()
+    except Exception:
+        raise RuntimeError("Unable to find package info.")
+
+    # The version line must have the form
+    # __version__ = 'ver'
+    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
+                              info_file, re.M)
+    author_match = re.search(r"^__author__ = ['\"]([^'\"]*)['\"]",
+                             info_file, re.M)
+
+    if version_match and author_match:
+        return version_match.group(1), author_match.group(1)
+    raise RuntimeError("Unable to find package info.")
+
+
+def _validate_install_ansible_base():
+    """Validate that we can install ansible-base. Currently this only
+    cares about upgrading to ansible-base from ansible<2.10
+    """
+    if os.getenv('ANSIBLE_SKIP_CONFLICT_CHECK', '') not in ('', '0'):
```

Comment:
Hmmn... I think we are going to want the env var here and in the ansible package to match.  Otherwise users will have to set both if they want to override.  If you prefer your name, just open a pr against ansible-community/antsibull to change that one

Reply:
That was the plan. We discussed the env var, and preferred this one. Once this merged I planned to do as you mention.

Reply:
<nod> please open that pr sooner, rather than later (aka, as soon as you are reading certain this is the env var name you want.). It is user facing so it would be good to expose the minimal number of people possible to the old name